<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefDialog</class>
 <widget class="QDialog" name="PrefDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="md1Layout">
       <item>
        <widget class="QLabel" name="md1Label">
         <property name="text">
          <string>MajDeduct Lv1 (restart) value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="md1SpinBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="md2Layout">
       <item>
        <widget class="QLabel" name="md2Label">
         <property name="text">
          <string>MajDeduct Lv2 (switch) value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="md2SpinBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="md3Layout">
       <item>
        <widget class="QLabel" name="md3Label">
         <property name="text">
          <string>MajDeduct Lv3 (detach) value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="md3SpinBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="multiClickLayout">
       <item>
        <widget class="QLabel" name="multiClickLabel">
         <property name="text">
          <string>Multi-click value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="multiClickSpinBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="totalTimeLayout">
       <item>
        <widget class="QLabel" name="totalTimeLabel">
         <property name="text">
          <string>Freestyle length (seconds)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="totalTimeSpinBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="timerDigitsLayout">
       <item>
        <widget class="QLabel" name="timerDigitsLabel">
         <property name="text">
          <string>Timer digits for partial seconds</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="timerDigitsSpinBox"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PrefDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PrefDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
